src/App.tsx

```diff
@@ -1,93 +1,93 @@
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useState } from 'react';
+import { Todos } from './Todos';
+import './styles/form.scss';
+import { client } from './utils/fetchClient';
+import { FormUser } from './FormUser';
 
-export const App: React.FC = () => {
-  return (
-    <div className="todoapp">
-      <header className="header">
-        <h1>todos</h1>
+export enum ErrorInput {
+  NONE,
+  EMAILERROR,
+  NAMEERROR,
```

Comment:
```suggestion
  EMAIL,
  NAME,
```
Error is in the enum name - so it's clear that it's an error - DRY